lmdeploy/model.py

```diff
@@ -1975,6 +1975,41 @@ def match(cls, model_path: str) -> Optional[str]:
             return 'molmo'
 
 
+@MODELS.register_module(name='llama4')
+class Llama4(BaseChatTemplate):
```

Comment:
May add UT in test_model.py, checking if the result of `message2prompt` equals to `apply_chat_template`